src/IW/Db/Repo.hs

```diff
@@ -43,3 +45,14 @@ upsertRepos = executeMany [sql|
         descr = EXCLUDED.descr
       , categories = EXCLUDED.categories;
 |]
+
+-- | Update a repo's categories field.
+updateRepoCategories :: (MonadCabal m, WithDb env m) => Repo -> m ()
+updateRepoCategories Repo{..} = do
+    categories <- getCabalCategories repoOwner repoName
+    execute [sql|
+        UPDATE repos
+        SET categories = ?
+        WHERE
+            (owner, name) = (?, ?)
```

Comment:
Instead of `(x, y) = (?, ?)` it's better to use `AND` operator. It's more idiomatic SQL than using tuples in SQL:

```haskell
WHERE owner = ?
  AND name = ?
```